Userland/Libraries/LibGfx/ICC/Profile.cpp

```diff
@@ -1588,9 +1219,9 @@ Crypto::Hash::MD5::DigestType Profile::compute_id(ReadonlyBytes bytes)
     const u8 zero[16] = {};
     Crypto::Hash::MD5 md5;
     md5.update(bytes.slice(0, 44));
-    md5.update(ReadonlyBytes { zero, 4 }); // profile flags field
+    md5.update(ReadonlyBytes { zero, 4 });  // profile flags field
```

Comment:
linter is not happy about this whitespace change

Reply:
different clang-format versions here than remotely, i suppose :/

Reply:
`Ubuntu clang-format version 15.0.7` on CI, which generally tries to use the latest stable version that is not a total pain to install on ubuntu - if you're using something newer which matches this criteria, please lmk!